### PR TITLE
Updates to rust libraries

### PR DESCRIPTION
**Pull request by untitaker**
cc @keats

### CODE REVIEW

File: views/libraries/rust.jade [hunk 1]
```diff
@@ -74,11 +74,11 @@ article.jwt-rust.accordion(data-accordion)
...
+        span.stars(data-repo='GildedHonour/rust-jwt', style='display: inline;')
           i.icon-budicon-466
       .repository
         i.icon-1392070209-icon-social-github
         a(href='https://github.com/GildedHonour/rust-jwt') View Repo
 
     .panel-footer
-      code cargo: name = "jwt" version = "*"
\ No newline at end of file
+      code cargo: name = "jwt" version = "*"
```
Comment (Keats) [inline views/libraries/rust.jade]: jwt points to https://github.com/mikkyang/rust-jwt, not GildedHonour/rust-jwt

Comment (untitaker) [inline views/libraries/rust.jade]: Nice, another inaccuracy in master :(

File: views/libraries/rust2.jade [hunk 2]
```diff
@@ -0,0 +1,84 @@
+// Rust GildedHonour/rust-jwt
```
Comment (Keats) [inline views/libraries/rust2.jade]: Keats/rust-jwt ?

**Comment by untitaker** [comment]
Updated.

**Comment by Keats** [comment]
lgtm

**Comment by untitaker** [comment]
bump?

**Pull request merged**